src/components/RescueDetails.tsx

```diff
@@ -156,17 +157,49 @@ export default function RescueDetails({id}: { id: string }) {
                 setError('Failed to update bird status. Please try again.')
             }
 
-            router.push("/")
+            // router.push("/")
```

Comment:
can we just delete this?

Reply:
Yeah I will remove this, I wasn't sure if we still wanted to route to the homepage after the status is changed or not. It probably would be a better UX to not route to the homepage, since everything gets updated via state anyways on the detail page.